<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employees' performance</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>891</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="salesPerformance">
    <attribute name="title">
     <string>Sales Performance</string>
    </attribute>
    <widget class="QPushButton" name="query_button">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Do query</string>
     </property>
    </widget>
    <widget class="QComboBox" name="employees_cb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose employee</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="sales_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>801</width>
       <height>381</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>20</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QGroupBox" name="radio_group">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Period</string>
     </property>
     <widget class="QRadioButton" name="monthly_radio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Monthly</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="quarterly_radio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Quarterly</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="graph">
    <attribute name="title">
     <string>Graph</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
